server/channels/app/upload.go

```diff
@@ -230,7 +230,9 @@ func (a *App) UploadData(c request.CTX, us *model.UploadSession, rd io.Reader) (
 			return nil, err
 		}
 		if written < minFirstPartSize && written != us.FileSize {
-			a.RemoveFile(uploadPath)
+			if err2 = a.RemoveFile(uploadPath); err2 != nil {
```

Comment:
You need to define `err2`
```suggestion
			if err2 := a.RemoveFile(uploadPath); err2 != nil {
```